bigframes/ml/llm.py

```diff
@@ -892,8 +892,12 @@ def fit(
         Returns:
             GeminiTextGenerator: Fitted estimator.
         """
-        if self._bqml_model.model_name.startswith("gemini-1.5"):
-            raise NotImplementedError("Fit is not supported for gemini-1.5 model.")
+        # Support gemini-1.5 and gemini-pro
+        supported_models = ["gemini-pro", "gemini-1.5-pro-002", "gemini-1.5-flash-002"]
+        if self.model_name not in supported_models:
+            raise NotImplementedError(
+                "Score is not supported models other than gemini-pro or gemini-1.5 model."
```

Comment:
This is in fit() in stead of score().

Reply:
I am working on two bugs,  b/381936588 and b/344891364, both fit() and score() needed to be updated.

Reply:
I mean the message is incorrect

Reply:
done